sass/compat/themes/minimog/_woocommerce-form-fields.scss

```diff
@@ -0,0 +1,11 @@
+// ------------------------------------
+// WOOCOMMERCE FORM-ROW
+// ------------------------------------
+
+body#{ $_body-theme-selector } .woocommerce-checkout .fc-wrapper .woocommerce-shipping-fields .woocommerce-shipping-fields__field-wrapper {
+	display:block;
+}
+
+ 
+ 
+ 
```

Comment:
Unnecessary empty lines, should only have one extra space at the end of the file